docs/cloud/billing.md

```diff
@@ -17,19 +17,19 @@ If a project is added during the billing cycle then a new charge is made for a f
 
 ## Removing Projects
 
-If you delete a project but still have other projects on their account you will receive a pro-rated credit for the number of whole days remaining in the billing cycle, which is used against a future invoice.
-If you delete your final project you will receive a pro-rated refund for the number of days remaining.
-The minimum cost for a project that is created and then deleted will be one day.
+If you delete a project but still have other projects on their account you will receive a pro-rated credit for the number of seconds remaining in the billing cycle, which is used against a future invoice.
```

Comment:
```suggestion
When a project is deleted you will receive a pro-rated credit for the number of seconds remaining in the billing cycle, which is used against a future invoice.
```

Reply:
I got confused by the if statement, what happens if the team is empty?

Reply:
there's currently no difference if its the last project on the team or there are others, the account gets a credit and it sits on the account until there's an invoice.
We can manually  refund credit if they close their account and request it